src/components/pages/notifier/myoffers/NotifierMyOffersPage.tsx

```diff
@@ -125,11 +128,14 @@ const NotifierMyOffersPage: FC = () => {
     }
   }, [account, web3, reportError])
 
+  const editProvider = (): void => {
+    history.push(ROUTES.NOTIFIER.MYOFFERS.EDIT)
+  }
+
   const handleAddPlan = logNotImplemented('handle add plan')
   const handleEditPlan = logNotImplemented('handle edit plan')
   const handleCancelPlan = logNotImplemented('handle cancel plan')
-  const handleEditProfile = logNotImplemented('handle edit profile')
-
+  const handleEditProfile = editProvider
```

Comment:
I think we could do this directly here
```suggestion
  const handleEditProfile = ():void => {
    history.push(ROUTES.NOTIFIER.MYOFFERS.EDIT)
  }
```